pages/lists/[listId]/activity.tsx

```diff
@@ -167,17 +131,14 @@ const ListActivityPage = ({
         <Error errorMessage="Unable to load list activity" />
       ) : (
         <div className="grid grid-cols-1 xl:grid-cols-2 gap-4">
-          <ClientOnly>
```

Comment:
I removed the client-only component as it's not required anymore as data only loads client-side now for graphs.